variants/MAPLEMINI_F103CB/variant.h

```diff
@@ -108,8 +108,9 @@ extern "C" {
 #define PIN_SERIAL_RX           PA10
 #define PIN_SERIAL_TX           PA9
 
-// USB
-#define USB_DISC_PIN            PB9
+// USB, pull this pin low to enable the USB attach pullup
+#define USBD_ATTACH_PINNAME     PB_9
+#define USBD_ATTACH_LEVEL       LOW
```

Comment:
I'm not entirely sure how this works, but shouldn't one of these be low and the other high?

Reply:
What do you mean? The PINNAME indicates the pin to control the pullup, and the LEVEL indicates the level it needs to have to ATTACH. What should be HIGH here? Also see the comments in variants/board_template/variant.h.